src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/QueueIntegrationTest.kt

```diff
@@ -136,7 +138,8 @@ abstract class QueueIntegrationTest : IntegrationTest() {
 
   fun loadPrisoners(prisoner: List<PrisonerBuilder>) {
     setupIndexes()
-    val prisonerNumbers = prisoner.map { it.prisonerNumber }.toList()
+    val prisonerNumbers = prisoner.map { it.prisonerNumber }
+    assertThat(prisonerNumbers.groupingBy { it }.eachCount().filter { it.value != 1 }).hasSize(0)
```

Comment:
something to stop accidentally adding multiple prisoners with same prisoner number